devbox.go

```diff
@@ -367,16 +373,27 @@ func (d *Devbox) PrintShellEnv() error {
 	return nil
 }
 
-func (d *Devbox) Info(pkg string) error {
+func (d *Devbox) Info(pkg string, markdown bool) error {
 	info, hasInfo := nix.PkgInfo(d.cfg.Nixpkgs.Commit, pkg)
 	if !hasInfo {
 		_, err := fmt.Fprintf(d.writer, "Package %s not found\n", pkg)
 		return errors.WithStack(err)
 	}
-	if _, err := fmt.Fprintf(d.writer, "%s\n", info); err != nil {
+	if _, err := fmt.Fprintf(
```

Comment:
If not markdown, what does it print? (do we need the markdown option?)

Reply:
For packages that don't have the configuration we just print the exact version you get (which is all that nix provides)